upup/models/cloudup/resources/addons/networking.projectcalico.org/pre-k8s-1.6.yaml.template

```diff
@@ -70,6 +70,8 @@ spec:
       labels:
         k8s-app: calico-node
         role.kubernetes.io/networking: "1"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
I think pre 1.6 uses a different API. Toleration did not have there own API values. Can you test on 1.5? I think I am correct.